Jenkinsfile

```diff
@@ -59,5 +62,8 @@ pipeline {
         onlyStable: false,
         zoomCoverageChart: false
     }
+    cleanup{
+        deleteDir()
```

Comment:
how about 
```
dir('node_modules') {
  deleteDir()
}
```
This will delete most of MB but will not require full checkout on the next run

Reply:
👍